packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/oktatookta/appatidp.md

```diff
@@ -12,6 +12,8 @@ When you are configuring federation between two Okta orgs using OpenID Connect:
     * Needs to be a secure domain that you own
     * Should match the redirect URI sent in the authorize request from the client
 
+    > **Note:** If you set `issuerMode` to `DYNAMIC` [mode](/docs/reference/api/idps/#property-details), then you must add both the Okta URI and the custom URI in the external IdP's allowed `redirectUri` list.
+
```

Comment:
Same comment as above. I'm so sorry you had to add this note for every IdP appatidp.md snippet! One of the many downsides of using this feature :-(